Wikipedia/Code/ReadingListsAlertController.swift

```diff
@@ -68,4 +68,25 @@ public class ReadingListsAlertController: NSObject {
         actions.forEach { alert.addAction($0) }
         return alert
     }
+    
+    // MARK: - ScrollableEducationPanelViewController presentation
+    
+    @objc func showLimitHitForDefaultListPanelIfNecessary(presenter: UIViewController, dataStore: MWKDataStore, readingList: ReadingList, theme: Theme) {
+        guard Thread.isMainThread, dataStore.readingListsController.isSyncEnabled else {
```

Comment:
Consider moving the main thread check to its own guard statement with a `assertionFailure("Expected main thread")` so we'll know right away if this ever happens on an unexpected thread.